### PR TITLE
E2e improve

### PR DESCRIPTION
**Pull request by leonwanghui**
**What this PR does / why we need it**:
Fix some bugs and reduce useless code.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by leonwanghui** [comment]
@xing-yang @wisererik Please take a review at this patch, thanks!

File: test/e2e/connector/connector.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```
Comment (xing-yang) [inline test/e2e/connector/connector.go]: 2018?

File: test/e2e/connector/connector.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/opensds/opensds/contrib/connector"
+	_ "github.com/opensds/opensds/contrib/connector/iscsi"
```
Comment (xing-yang) [inline test/e2e/connector/connector.go]: If this is not used, why importing it?  Just to test the import path is correct?

File: test/e2e/connector/connector.go (hunk 1 above)
Comment (leonwanghui) [reply test/e2e/connector/connector.go]: Got it

File: test/e2e/connector/connector.go (hunk 2 above)
Comment (leonwanghui) [reply test/e2e/connector/connector.go]: No, this import is used for setting up `init()` method in iscsi package.

**Pull request merged**